src/pytest/test_issues.py

```diff
@@ -0,0 +1,17 @@
+def test_1282(env):
+  env.expect('FT.CREATE idx SCHEMA txt1 TEXT txt2 TAG').equal('OK')
+  env.expect('FT.ADD idx doc1 1.0 FIELDS txt1 foo txt2 bar').equal('OK')
+  env.expect('FT.ADD idx doc2 1.0 FIELDS txt1 baz txt2 xyz').equal('OK')
+  env.expect('FT.ADD idx doc3 1.0 FIELDS txt1 hello txt2 world').equal('OK')
+  env.expect('FT.ADD idx doc4 1.0 FIELDS txt1 jim txt2 burie').equal('OK')
+
+
+  # test with tags would crash server
+  env.expect('FT.SEARCH idx', '~world ~bar')
+
+#  res = [4L, 'doc3', ['txt1', 'hello', 'txt2', 'world'],
```

Comment:
Remove?

Reply:
We should fix it next.
Thought of it as TODO...